src/frontend/js/components/SearchFilterGroupModal/index.tsx

```diff
@@ -257,8 +241,13 @@ export const SearchFilterGroupModal = ({ filter }: SearchFilterGroupModalProps)
       <Modal
         appElement={modalExclude}
         bodyOpenClassName="has-search-filter-group-modal"
-        className="search-filter-group-modal modal--stretched"
+        className="search-filter-group-modal"
         isOpen={modalIsOpen}
+        title={
+          <h2>
+            <FormattedMessage {...messages.modalTitle} values={{ filterName: filter.human_name }} />
+          </h2>
```

Comment:
you can use the `intl.formatMessage` to retrieve a string
```suggestion
title={intl.formatMessage({...messages.modalTitle, values:{ filterName: filter.human_name }})}
```